src/rocksdb.rs

```diff
@@ -1301,6 +1314,22 @@ mod test {
         assert!(batch.data_size() > prev_size);
         batch.clear();
         assert_eq!(batch.data_size(), prev_size);
+
+        // test save point
+        let mut batch = WriteBatch::new();
+        batch.put(b"k10", b"v10").unwrap();
+        batch.set_save_point();
```

Comment:
If I set save point multi times, if the new save point will override the old one?